tests/playwright/sanity/container.test.js

```diff
@@ -261,4 +261,56 @@ test.describe( 'Container tests', () => {
 			container: false,
 		} );
 	} );
+
+	test( 'Fallback image is not on top of background video', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+
+		await page.goto( '/wp-admin/media-new.php' );
+
+		if ( await page.locator( 'text=browser uploader' ).nth( 1 ).isVisible() ) {
```

Comment:
nth is not the best option
also, try to avoid search by text if possible
can you extract the element to a const ?

Reply:
@TzviRabinovitch What is your consideration to avoid text? For RTL reasons or other?

Reply:
@TzviRabinovitch Done.

Reply:
@hein-obox yes